Simplenote/src/main/java/com/automattic/simplenote/NoteWidgetConfigureActivity.java

```diff
@@ -152,9 +192,9 @@ public void onClick(View view) {
                     PendingIntent pendingIntent = PendingIntent.getActivity(context, mAppWidgetId, intent, 0);
 
                     // Set widget content
-                    views.setOnClickPendingIntent(R.id.widget_layout, pendingIntent);
-                    views.setTextViewText(R.id.widget_text, note.getTitle());
-                    widgetManager.updateAppWidget(mAppWidgetId, views);
+                    mRemoteViews.setOnClickPendingIntent(R.id.widget_layout, pendingIntent);
+                    mRemoteViews.setTextViewText(R.id.widget_text, note.getTitle());
```

Comment:
This `setTextViewText` is called here but the color is not updated. 

When you create a new widget the text color remains `simplenote_light_grey` from line 137 of  `NoteWidget.java`. It's correctly updated after opening the app.

I think this is the only place where that can happen

Reply:
I added `setTextViewText` to `NoteWidgetConfigureActivity` in 72ae01c.

Reply:
Fix confirmed!